extensions/amp-story/1.0/amp-story.css

```diff
@@ -40,6 +40,15 @@ amp-consent {
   z-index: initial !important;
 }
 
+@media (prefers-reduced-motion: reduce) {
+  *,
```

Comment:
Isn't this going to be a super expensive selector?

Reply:
I'm not entirely sure, I'm never really worried about CSS selectors usually, only about their properties.
I got most of the information here: https://developers.google.com/web/updates/2019/03/prefers-reduced-motion#bonus_forcing_reduced_motion_on_all_websites

Reply:
(There's no obvious difference in the metrics the devtools give, but I didn't check the evaluation time)

Reply:
Sounds good, but worth keeping in mind in case we notice a difference between regular and reduced motion.  I was primarily worried because stories often have long and deep DOM trees